include/openthread/platform/radio.h

```diff
@@ -315,6 +316,20 @@ typedef struct otRadioFrame
              */
             uint8_t mRxChannelAfterTxDone;
 
+            /**
+             * The transmit power in dBm.
+             *
+             * If the platform layer does not provide `OT_RADIO_CAPS_TRANSMIT_POWER` capability, it can ignore this
+             * value.
+             *
+             * If the value is 127, then the platform should ignore this value and transmit the frame with its default
```

Comment:
What if the value of `mTxPower` is larger than the value we set via `otPlatRadioSetChannelMaxTransmitPower()` or `otPlatRadioSetChannelTargetPower()`?